resources/factions/USA 1970 Vietnam War.json

```diff
@@ -76,7 +76,8 @@
   ],
   "naval_units": [
     "DDG Arleigh Burke IIa",
-    "CV-59 Forrestal",
+    "CVA-31 Bon Homme Richard",
+    "CV-59 USS Forrestal",
```

Comment:
Should be changed back to "CV-59 Forrestal" (unit names in a faction file always reference the name that's specified within the unit's yaml file)